com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -457,19 +520,6 @@ var Progress = {
     progress_id: "defold-progress",
     bar_id: "defold-progress-bar",
 
-    listeners: [],
-
-    addListener: function(callback) {
```

Comment:
Left `Progress` object only for visualization.

Reply:
It's not that critical if we break something in templates, because we always can notify users and ask them to update their template (btw could you please a it into your PR message e.g. [like here](https://github.com/defold/defold/pull/8591)) but it would be nice to keep existing extensions compatible if it's possible. For example:

https://github.com/defold/extension-poki-sdk/blob/main/poki-sdk/manifests/web/engine_template.html

If it's impossible to avoid - sure, we have to notify users and probably introduce it as a breaking change, but here it feels like something only about naming. I suggest to keep `Progress` for logic and create `ProgressView` just for you, to keep it back compatible

Reply:
I left Progress object as wrapper around new ProgressView and ProgressUpdater. Let me know if such approach is ok.